ts/features/itwallet/discovery/screens/ItwDiscoveryInfoScreen.tsx

```diff
@@ -23,9 +28,16 @@ import { ItwEidIssuanceMachineContext } from "../../machine/provider";
  * with a primary and secondary action.
  */
 const ItwDiscoveryInfoScreen = () => {
+  useOnFirstRender(trackItWalletIntroScreen);
```

Comment:
Screen views are focus events. Using `useOnFirstRender` will track only the first render, when the screen is put in the navigation stack. So, if the user return back, the screen view is not correctly tracked.

```suggestion
  useFocusEffect(trackItWalletIntroScreen);
```